RFC-010-SWAP-Omni-Layer.md

```diff
@@ -0,0 +1,117 @@
+# The Omni Layer Token Asset
+
+- RFC-Number: 010
+- Status: Draft
+- Discussion-issue: -
+- Created on: 27 Mar. 2019
+
+## Table of Contents
+
+<!-- toc -->
+
+  * [Description](#description)
+  * [The Omni Layer Assets](#the-omni-layer-assets)
+    + [`quantity`](#quantity)
+    + [`property_id`](#property_id)
+  * [Registry extension](#registry-extension)
+- [Examples](#examples)
+
+<!-- tocstop -->
+
+## Description
+
+This RFC specifies how Bitcoin based Omni Layer Assets MUST be described in [RFC002](./RFC-002-SWAP.md) SWAP messages.
+OmniLayer refers specifically to [Omni Protocol](https://github.com/OmniLayer/spec).
+As required by [RFC002](./RFC-002-SWAP.md), this RFC defines the *asset* value `omni_layer`.
+This in combination with subsequent RFCs that define concrete execution steps for particular SWAP protocols enable users to negotiate and execute Omni Layer Token swaps.
```

Comment:
I'd prefer we follow the way I did ERC20:
1. Call the file RFC-010-Omni-Layer.md
2. Make the description stand alone without regards to SWAP messages (the asset definition could easily be used outside of SWAP messages). Just mention that:
```
The Asset type header was introduced in [RFC002](./RFC-002-SWAP.md) to describe assets being exchanged in a COMIT SWAP protocol.
```